### ISSUE TITLE
Add a page (resources?) where official logo image can be downloaded

### ISSUE THREAD

**Issue by nickynicolson** [tdwg/website#66]
I'm creating a set of slides for a conference outside of the biodiversity informatics community. To credit TDWG for standards development work, I'd like access to a TDWG logo - so it would be useful if there were a standard resource page where this could be downloaded. 
Currently I guess a lot of people just use google image search, ending up with things that are out of date or in poor resolution - or don't credit at all.

**Comment by dkoureas** [comment]
I now have a vector version of the TDWG logo and a .pdf and .ai version of a banner.

**Comment by stanblum** [comment]
Dimitris, can you upload or pass them to Peter?

**Comment by dkoureas** [comment]
[TDWG_Logos.zip](https://github.com/tdwg/website/files/2285971/TDWG_Logos.zip)

**Comment by dkoureas** [comment]
Color scheme for TDWG logo 
For dark background TDWG logo is filled CMYK 5/4/4/0 - RGB 244/244/244 - F4F4F4)
For light background TDWG logo is filled CMYK 100/72/40/34 - RGB 4/59/88 - 043B58

**Comment by MattBlissett** [comment]
I don't remember where I found it, but when I needed one I used this SVG.

<a href="http://mb.gbif.org/logo-tdwg.svg"><img src="http://mb.gbif.org/logo-tdwg.svg" width="500px" alt="TDWG logo"></a>

The font is different to Dimitris'.

**Comment by dkoureas** [comment]
I think we have a new svg version of the in the website now

Kind regards,
Dimitris

*--*

*Dr Dimitris Koureas, FLS*
*Programme Director - Department Head*
International Biodiversity Infrastructures
Naturalis Biodiversity Center, P.O. Box 9517, 2300 RA Leiden, NL

Coordinator, Distributed System of Scientific Collections (DiSSCo
<http://dissco.eu/>)
Chair, Biodiversity Information Standards Organisation (TDWG
<http://tdwg.org/>)
Research Data Alliance (RDA <http://www.rd-alliance.org/>) - Technical
Advisory Board member

*ORCID*: 0000-0002-4842-6487 | *Linkedin*: linkedin.com/in/dkoureas
*Twitter*: @DimitrisKoureas | *Tel*: +31 (0) 71 751 9251

**Comment by MattBlissett** [comment]
In https://github.com/tdwg/website/commit/18e4271d0b169345f691a02c9dfa4c85037245a4 I've converted the two-colour logo with the correct font into the monochrome versions Dmitris provided. See http://labs.gbif.org/~mblissett/2018/12/tdwg_logos/tdwg-logos.html for a quick look at the images.

Unless the font has been changed intentionally, the logos Dmitris has were exported without the appropriate font being available.

I chose nicer widths (1360px, 1600px) so the PNG images can scale better if people resize them to common pixel sizes. The SVGs should be preferred wherever possible, as vector files they're both smaller and infinitely resizable without going fuzzy.
